tests/test_interpolate.py

```diff
@@ -351,3 +352,623 @@ def test_duplicate_idxs_period_pairs():
     assert duplicate_torsion_idxs not in counts_kv_dst
 
     st.setup_intermediate_state(0.5)
+
+
+def test_padded_interpolation():
+    # verify that symmetrically padded interpolation has the correct symmetry, staggering, truncating,
+    # and interpolation behavior.
```

Comment:
nit: currently this only seems to test that we correctly return the end-state values outside of the interpolation range?

Reply:
gah, good catch, another stale comment.